crud/users.py

```diff
@@ -21,13 +21,69 @@ async def create_user(user: users_schema.UserCreate):
     if db_user:
         raise HTTPException(status_code=400, detail="Email already registered")
 
+    query = users_model.user.select()
+    stored_data = await database.fetch_one(query)
+    if stored_data == None: is_admin = True
+    else: is_admin = False
+
     salt = users_util.get_random_string()
     hashed_password = users_util.hash_password(user.password, salt)
-    query = users_model.users_table.insert().values(
-        email=user.email, name=user.name, hashed_password=f"{salt}${hashed_password}"
+    query = users_model.user.insert().values(
+        email=user.email, name=user.name, hashed_password=f"{salt}${hashed_password}",
+        is_admin=is_admin
     )
     user_id = await database.execute(query)
     token, expire = users_util.create_access_token(user.email)
     token_dict = {"token": token, "expires": expire}
 
-    return {**user.dict(), "id": user_id, "is_active": True, "token": token_dict}
+    return {**user.dict(), "id": user_id, "is_admin": is_admin, "is_active": True, "token": token_dict}
+
+
+async def filter_users(filter: users_schema.UserFilter, offset: int = 0, limit: int = 100):
```

Comment:
filter can be optional

Reply:
even without blank json/dict? Ok, will add default blank value.